### DISCUSSION TITLE
第4节课笔记(15班) 

### DISCUSSION THREAD

**Discussion by ironartisan** [InternLM/Tutorial#39] [笔记]
请在下方提交笔记链接

**Comment by dqyydq** [comment]
https://blog.csdn.net/qq_52410386/article/details/135544960?spm=1001.2014.3001.5501

**Comment by IyGnahz** [comment]
笔记+作业 https://g6xuom3zbh.feishu.cn/docx/G8szdsIcXoVlPkxLe3HcwKfJnRe

**Comment by seienn** [comment]
https://gvxxiiwsufy.feishu.cn/wiki/IaNJwfZzQiNd6Jk3N3yc7SXPnYb?from=from_copylink

**Comment by szhowardhuang** [comment]
这次内容太多了。两天才学习完成，一边学一边做笔记。提供pdf如下：

[Finetune 简介.pdf](https://github.com/InternLM/tutorial/files/13918273/Finetune.pdf)

**Comment by lizhunkg** [comment]
第四节课笔记： https://note.youdao.com/s/9YqolG2T

**Comment by scchy** [comment]
第四节课笔记：https://github.com/scchy/My_Learn/blob/master/LLM_prime/4_InternLM_Xtun_doc.md

**Comment by woodx9** [comment]
请参考笔记部分 [飞书链接](https://gxloh9mfvxh.feishu.cn/wiki/LsKqwSpWviujS8kh88KcRAlzn9g?from=from_copylink)

**Comment by sokolo05** [comment]
第四节课笔记： https://github.com/sokolo05/Scholar_PuYu/blob/main/02.%E8%AF%BE%E7%A8%8B%E7%AC%94%E8%AE%B0/%E7%AC%AC4%E8%8A%82_%E7%AC%94%E8%AE%B0.md

**Comment by ziptoam** [comment]
第四节课笔记
https://note.youdao.com/s/YQkqcnT8

**Comment by 520Enterprise** [comment]
https://www.cnblogs.com/520Enterprise/p/17945399

**Comment by yabgdsajghsj** [comment]
https://note.youdao.com/s/UkpEZvjX

**Comment by Gaia-yue** [comment]
https://github.com/Gaia-yue/---/blob/main/LLM/4_finetune-note.md

**Comment by Mrguanglei** [comment]
https://mqf0lvyk616.feishu.cn/wiki/WB1jwzwBDigfuZkz4bjcV9tPnQf?from=from_copylink

**Comment by bblgbr** [comment]
不好意思 晚提交了，第四节课笔记：https://github.com/bblgbr/play_llm/blob/main/InterLM_Class/04%20XTuner%E5%A4%A7%E6%A8%A1%E5%9E%8B%E5%8D%95%E5%8D%A1%E5%BE%AE%E8%B0%83.md

**Comment by NANDSS1** [comment]
https://www.yuque.com/nandss/gyyg9p/em2kt27koibwnelm?singleDoc# 《lesson4 XTuner单卡低成本微调实战（笔记）》

**Comment by g29times** [comment]
https://novagen.notion.site/4-Finetune-1ae55218a51d4856abcb50d0e4f87410?pvs=4
